README.md

```diff
@@ -101,7 +102,7 @@ ssl_verify_mode = ":verify_peer"
 
 [automate]
 enable = false
-url = "https://<automate_url>"
+server_url = "https://<automate_url>/data-collector/v0/"
```

Comment:
Why the name change? Will this break existing installs who have set where to send data to Automate?

Reply:
the name change is to fix a bug where the `client-config` was looking for `cfg.automate.server_url`.

Reply:
So no existing configurations using `cfg.automate.url` work at the moment anyway?

Reply:
Correct that was a mistake I made I updated the logic but didn't update the readme.